index.html

```diff
@@ -198,5 +198,23 @@ <h2 class="aboutLink">LET'S CONNECT</h2>
             </ul>
         </section>
     </section>
+
+    <section class="contactMe">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/mbjeoppw" method="post">
+            <ul>
+                <li>
+                    <input name="user_name" id="name" placeholder="Yeremias" type="text" maxlength="30" required>
+                </li>
+                <li><input name="email" id="email" placeholder="natajayanj14@gmail.com" type="email" required>
+                </li>
+                <li>
```

Comment:
- [ ] This is an example from your `index.html` where the used indentation(the space preceding each new line of code) is set to 4 spaces, which is not a good practice, please make sure to use 2 spaces for your default indentation as it improves readability, please refer to this [lesson](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for more info.